docs/DeveloperGuide.md

```diff
@@ -509,15 +512,19 @@ Step 1: The user accesses the PatientSync application.
 
 Step 2: The user executes the `adde 1 n/ Birthday d/ 20-01-2022` command to add the Event, Birthday,
 which falls on the 20th January.
+
 * Upon successful validation, it creates an `AddEventsCommand` instance.
 
-Step 3: The use executes `deletee 1 e/1` to delete the Event as he realised he keyed in the wrong
-date.
+Step 3: The use executes `deletee 1 e/1` to delete the Event as the event is over.
+
 * Upon successful validation,  an `DeleteEventCommand` instance is created.
 
 The following UML sequence diagram illustrates how the Delete Event operation works.
 <puml src="diagrams/DeleteEventSequenceDiagram.puml" alt="Delete Event Sequence Diagram" />
 
+**Note:** The lifeline for `DeleteEventCommandParser` and `DeleteEventCommand` should end at the destroy marker (X) but
+due to a limitation of PlantUML, the lifeline reaches the end of diagram.
```

Comment:
Nice catch on this note! Is this an issue that occurs for the other Sequence Diagrams as well?

Reply:
Yes! This is something that the other sequence diagram needs to add as well!